static/js/components/NotificationRow.tsx

```diff
@@ -24,11 +24,31 @@ const NotificationRow: FC<Props> = ({ notify }) => {
   if (!notify.notification) {
     return null;
   }
-  const { type, message } = notify.notification;
+  const { actions, title, type, message } = notify.notification;
+  const figureTitle = () => {
+    if (title) {
+      return title;
+    }
+    switch (type) {
+      case "negative":
+        return "Error";
+      case "positive":
+        return "Success";
+      case "information":
+        return "Info";
+      default:
+        return null;
```

Comment:
I'd return an empty string to stick to the same type. And below, when you call it, you could do:
```
title={figureTitle() ?? undefined}
```

Reply:
That is not valid typescript, because empty string can't be used as a left-hand side of the `??` chain.

Reply:
Sure, you're right. `title={figureTitle() || undefined}` then.

Reply:
right, that works and is better.